mgmtsystem_nonconformity_quality_control_oca/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
```

Comment:
```suggestion
```

nitpicking: there's no need to put headers on __init__.py files

Reply:
ok, copy that